src/API/WesternStatesWater.WestDaat.Contracts.Client/Requests/Conservation/EstimateEvapotranspirationRequestValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FluentValidation;
+
+namespace WesternStatesWater.WestDaat.Contracts.Client.Requests.Conservation;
+
+public class EstimateEvapotranspirationRequestValidator : AbstractValidator<EstimateEvapotranspirationRequest>
+{
+    public EstimateEvapotranspirationRequestValidator()
+    {
+        RuleFor(x => x.FundingOrganizationId).NotEmpty();
+
+        RuleFor(x => x.WaterConservationApplicationId); // optional
+
+        RuleFor(x => x.Polygons).NotEmpty();
+        RuleForEach(x => x.Polygons).ChildRules(polygonEntryValidator =>
+        {
+            polygonEntryValidator.RuleFor(polygon => polygon).NotEmpty();
+        });
+
+        RuleFor(x => x.Model).NotEmpty();
+
+        RuleFor(x => x.DateRangeStart).NotEmpty();
+        RuleFor(x => x.DateRangeEnd).NotEmpty();
```

Comment:
Optional - you can chain (if you want to)

```suggestion
        RuleFor(x => x.DateRangeEnd)
            .NotEmpty()
            .GreaterThanOrEqualTo(x => x.DateRangeStart);
```

Reply:
smart. I should probably also apply the LessThanOrEqualTo check on the DateRangeStart to match